### PR TITLE
[over.best.ics] Turn a rationalizing footnote into a note with an acc…

### LINKED ISSUE #1152: Comments in code example in footnotes should be italic like elsewhere

### PR DESCRIPTION
**Pull request by jensmaurer**
…ompanying example.

Fixes #1152.

### CODE REVIEW

File: source/overloading.tex [hunk 1]
```diff
@@ -1937,59 +1937,27 @@
...
-were to be declared then
-\tcode{f(b)}
-would resolve to that
-\tcode{f(B)}
-because the exact match
-with
-\tcode{f(B)}
-is better than any of the sequences required to match
-\tcode{f(A)}.}.
+other user-defined conversion sequence.  \begin{note}
```
Comment (tkoeppe) [inline source/overloading.tex]: What do you think of making the note+example a new paragraph?

File: source/overloading.tex [hunk 2]
```diff
@@ -1937,59 +1937,27 @@
...
+ambiguous~(\ref{dcl.init}).
+\end{note} \begin{example}
+\begin{codeblock}
+class B;
+class A { A(B&); };
+class B { operator A(); };
+class C { C(B&); };
+B b;
+int g(A);
+int x = g(b);   // error: selects g(A), but converting the argument is ambiguous (\ref{over.match.copy})
```
Comment (Eelis) [inline source/overloading.tex]: Should g(A) have some \tcode here?

Comment (jensmaurer) [inline source/overloading.tex]: Yes.

File: source/overloading.tex (hunk 1 above)
Comment (jensmaurer) [inline source/overloading.tex]: The note + example definitely belong to the "ambiguous conversion sequence" paragraph. The example highlights the entire preceding paragraph, not just the note, so nesting is not an option.

I could make the example (but not the note) a new paragraph. It seems we recently gravitate towards making long code examples separate paragraphs. Done.

**Comment by jensmaurer** [comment]
Mooted by #1232 and commit 81eea5e1b5a477404b39176f71699f7b87a2dc01.

**Pull request closed without merging**